infoblox_client/objects.py

```diff
@@ -416,6 +418,9 @@ def fetch(self, only_ref=False):
 
     def update(self):
         update_fields = self.to_dict(search_fields='exclude')
+        fields = self.to_dict(search_fields='extra', update_fields=update_fields)
+        for key in fields:
+            LOG.info("Field is not allowed for update: %s - ignoring" % key)
```

Comment:
Please don't format log messages when calling log methods. Instead pass the format message and parameters as log method arguments, like this:
```suggestion
            LOG.info(
                "Field is not allowed for update: %s - ignoring",
                key,
            )
```

This is called lazy logging. This way, your message will be formatted only if the log level is `info`, or more verbose.